pkg/rancher-components/src/components/Form/Radio/RadioButton.vue

```diff
@@ -155,6 +162,12 @@ export default Vue.extend({
           {{ description }}
         </template>
       </div>
+      <div
+        v-else-if="hasDescriptionSlot"
+        class="radio-button-outer-container-description"
+      >
+        <slot name="description" />
```

Comment:
I'm not sure the conditional logic is needed - wouldn't it still work if line 169 was the only line added? I believe if you have a named slot that is not used, it just won't render.